Gemfile

```diff
@@ -23,6 +23,9 @@ gem 'mysql2'
 # for observing records
 gem 'rails-observers'
 
+# for rating
+gem 'ratyrate', github: 'wazery/ratyrate', branch: 'master'
```

Comment:
Something I am really **unsure** about. The released version of ratyrate contains a bug in the [rating_for_user](https://github.com/wazery/ratyrate#other-helpers) method that shows a particular user's rating. This method is required for blind voting. However, the master is quite stable and does not contain the bug. This could be a temporary change until we get a response regarding https://github.com/wazery/ratyrate/issues/125 ( issue asking for a new version release )